### PR TITLE
Correcting AdHoc Build Errors for Mac Notarization

### PR DESCRIPTION
**Pull request by johnmccrae**
Signed-off-by: John McCrae <john.mccrae@progress.com>

## Description
Mac builds have been failing with an error about invalid certificates. In tracing that back, Vikram found that the problem was a cleanup issue. I had commented out the ruby cleanup libraries while doing some other work but it turns out they are needed for cleaning up old certificates on test systems as well. This PR puts them back into the config\projects\chef.rb file

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -39,40 +45,6 @@ PATH
   remote: .
   specs:
     chef (18.0.142)
-      addressable
```
Comment (marcparadise) [inline Gemfile.lock:42]: This should probably not be getting removed, was `rake dependencies:update` used to update the lockfile?

**Review by marcparadise — CHANGES_REQUESTED** [review]
Could you also update the commit/PR description to include context/'why' for the change?

File: Gemfile.lock (hunk 1 above)
Comment (johnmccrae) [reply Gemfile.lock:42]: I did not specifically remove that. It came out during a bundle update?

Comment (johnmccrae) [reply Gemfile.lock:42]: It's still in the file, oddly. The PR just looks funny. If you look down to the bottom of that section you'll see the library still listed. It also shows up when you view the full file.

File: Gemfile.lock [hunk 2]
```diff
@@ -502,4 +478,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   2.3.18
+   2.3.21
```
Comment (vkarve-chef) [inline Gemfile.lock:481]: how about keeping the default bundler version 2.3.7 for now? to avoid the ruby-cleanup error.

**Review by tpowell-progress — APPROVED** [review]

**Comment by johnmccrae** [comment]
closing this as duplicated by tpowell work. His is merged already

**Comment by johnmccrae** [comment]
already corrected by a PR that tpowell merged

**Pull request closed without merging**